webpack.prod.js

```diff
@@ -1,8 +1,55 @@
 const merge = require("webpack-merge");
 const MinifyPlugin = require("babel-minify-webpack-plugin");
 const common = require("./webpack.common");
+const path = require("path");
+const config = require("./config");
+const HandlebarsTemplatePlugin = require("./templates/HandlebarsTemplatePlugin");
+const helpers = require("./templates/helpers");
 
 module.exports = merge(common, {
   devtool: "source-map",
-  plugins: [new MinifyPlugin()]
+  plugins: [
+    new MinifyPlugin(),
+    new HandlebarsTemplatePlugin({
+      pages: [
+        {
+          fileName: "index.html",
+          templatePath: path.resolve(__dirname, "templates", "index.hbs"),
+          data: {
+            baseAssetsPath: process.env.BASE_ASSETS_PATH,
+            config
+          }
+        },
+        {
+          fileName: "room.html",
+          templatePath: path.resolve(__dirname, "templates", "room.hbs"),
+          data: {
+            baseAssetsPath: process.env.BASE_ASSETS_PATH,
+            config
+          }
+        },
+        {
```

Comment:
note this should be `smoke-`, and I would probably suggest doing something more generic that will take a list of page names (`["index", "room"]`) and generically generate templates for `x.html` and `x-smoke.html`